src/main/java/org/gridsuite/study/server/repository/LoadFlowResultEntity.java

```diff
@@ -7,33 +7,47 @@
 
 package org.gridsuite.study.server.repository;
 
-import com.datastax.driver.core.DataType;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import org.springframework.data.cassandra.core.mapping.*;
+import lombok.*;
 
-import java.io.Serializable;
+import javax.persistence.*;
 import java.util.List;
 import java.util.Map;
 
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  * @author Etienne Homer <etienne.homer at rte-france.com>
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
  */
-@AllArgsConstructor
 @NoArgsConstructor
+@AllArgsConstructor
 @Getter
-@UserDefinedType("loadFlowResult")
-public class LoadFlowResultEntity implements Serializable {
+@Setter
+@Entity
+@Table(name = "loadFlowResult")
+public class LoadFlowResultEntity {
+
+    public LoadFlowResultEntity(boolean ok, Map<String, String> metrics, String logs, List<ComponentResultEntity> componentResults) {
+        this(null, ok, metrics, logs, componentResults);
+    }
+
+    @Id
+    @GeneratedValue(strategy  =  GenerationType.AUTO)
+    @Column(name = "id")
+    private Long id;
 
+    @Column(name = "ok")
     private boolean ok;
 
-    @CassandraType(type = DataType.Name.MAP, typeArguments = { DataType.Name.TEXT, DataType.Name.TEXT })
+    @Column(name = "metrics")
+    @ElementCollection(fetch = FetchType.EAGER)
```

Comment:
https://vladmihalcea.com/eager-fetching-is-a-code-smell/

Maybe we should make more effort to make lazy loading work.